scripts/bytecodecompare/prepare_report.py

```diff
@@ -470,6 +491,7 @@ def commandline_parser() -> ArgumentParser:
     generate_report(
         glob("*.sol"),
         Path(options.compiler_path),
+        options.execution_arch,
```

Comment:
Convert it to a proper enum value:
```suggestion
        ExecutionArchitecture(options.execution_arch),
```
argparse gives you a string and you have to convert it to actually get an enum. Then you don't have to unpack it with `value` on every use.

Reply:
thats great!